app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  def comments_counter
+    post.increament!(:comments_counter)
+  end
```

Comment:
- [ ] Kindly make use of the model callback functions such as `after_save` so that the counters are triggered every time a new comment is saved.

```ruby
  after_save :update_comment_counter

  private

  def update_comment_counter
    post.increment!(:comment_counter)
  end
```


![counters](https://github.com/derrick1451/Blog-app/assets/53124548/1dd768cb-0655-4e4f-9015-6f1f77aa42a0)